ext/standard/tests/general_functions/is_float_64bit.phpt

```diff
@@ -1,5 +1,5 @@
 --TEST--
-Test is_float() & it's FALIASes: is_double() & is_real() functions
+Test is_float() & it's FALIASes: is_double() functions
```

Comment:
nit: should be singular now, and "it's" is wrong anyway.
```suggestion
Test is_float() & its FALIAS is_double()
```

Reply:
Make sense! Fixed it :)